app/reducers/review.jsx

```diff
@@ -0,0 +1,119 @@
+import axios from 'axios'
+
+const initialState = {
+  reviews: [],
+  selectedReview: {}
+}
+
+/* -----------------    ACTION TYPES     ------------------ */
+const SET_REVIEWS = 'SET_REVIEWS'
+const SET_SELECTED_REVIEW = 'SET_SELECTED_REVIEW'
+const DELETE_REVIEW = 'DELETE_REVIEW'
+const UPDATE_REVIEW = 'UPDATE_REVIEW'
+//do we need a create_review?
+const CREATE_REVIEW = 'CREATE_REVIEW'
+const ADD_REVIEW_TO_ORDER = 'ADD_REVIEW_TO_ORDER'
+  // Below might not be needed
+  // I am leaving here till we have the order reducer built out.
+const UPDATE_REVIEW_IN_ORDER = 'UPDATE_REVIEW_IN_ORDER'
+const REMOVE_REVIEW_FROM_ORDER = 'REMOVE_REVIEW_FROM_ORDER'
+
+/* ------------   ACTION CREATORS     ------------------ */
+
+export const setReviews = (reviews) => ({
+  type: SET_REVIEWS,
+  reviews: reviews
+})
+
+export const setReview = (review) => ({
+  type: SET_SELECTED_REVIEW,
+  selectedReview: review
+})
+
+export const createReview = (review) => ({
+  type: CREATE_REVIEW,
+  review: review
+})
+
+export const updateReview = (review) => ({
+  type: UPDATE_REVIEW,
+  review: review
+})
+
+export const deleteReview = (reviewId) => ({
+  type: DELETE_REVIEW,
+  reviewId: reviewId
+})
+
+export const addReviewToOrder = (review) => ({
+  type: ADD_REVIEW_TO_ORDER,
+  review: review
+})
+
+export const removeReviewFromOrder = (review) => ({
+  type: REMOVE_REVIEW_FROM_ORDER,
+  review: review
+})
+
+export const updateReviewInOrder = (review) => ({
+  type: UPDATE_REVIEW_IN_ORDER,
+  review: review
+})
+
+
+/* ------------       REDUCERS     ------------------ */
+export default function(state = initialState, action) {
+  const newState = Object.assign({}, state)
+
+  switch (action.type) {
+    case SET_REVIEWS:
+      newState.reviews = action.reviews
+      break
+
+    case SET_SELECTED_REVIEW:
+      newState.selectedReview = action.selectedReview
+      break
+
+    case CREATE_REVIEW:
+      newState.reviews = newState.reviews.concat([action.reviews])
+      break
+
+    case UPDATE_REVIEW:
+      newState.reviews = newState.reviews.map((review) => (
+        (review.id === action.review.id) ? action.review : review
+      ))
+      break
+
+    case DELETE_REVIEW:
+      newState.reviews = newState.reviews.filter((currentReview) => (
+        (currentReview.id !== action.reviewId)
+      ))
+      break
+
+    case ADD_REVIEW_TO_ORDER:
```

Comment:
need to update all references to students. Made a new issue referencing this, disregard it here for now.